### ISSUE TITLE
coverage_format fails with an exception dart-sdk/lib/_internal/list.dart

### ISSUE THREAD

**Issue by zoechi** [dart-lang/coverage#53]
```
 (master) $ format_coverage --package-root=. -i coverage.json
Unhandled exception:
Uncaught Error: FileSystemException: Cannot resolve symbolic links, path = '/home/myuser/darteditor/dart-sdk/lib/_internal/list.dart' (OS Error: No such file or directory, errno = 2)
Stack Trace:
#0      FileSystemEntity._throwIfError (dart:io/file_system_entity.dart:671)
#1      FileSystemEntity.resolveSymbolicLinksSync (dart:io/file_system_entity.dart:338)
#2      Resolver.resolveSymbolicLinks (package:coverage/src/resolver.dart:66:62)
#3      Resolver.resolve (package:coverage/src/resolver.dart:44:34)
#4      PrettyPrintFormatter.format.<anonymous closure> (package:coverage/src/formatter.dart:55:33)
#5      Future.forEach.<anonymous closure>.<anonymous closure> (dart:async/future.dart:336)
#6      Future.Future.sync (dart:async/future.dart:168)
#7      Future.forEach.<anonymous closure> (dart:async/future.dart:336)
#8      Future.Future.sync (dart:async/future.dart:168)
#9      Future.doWhile.<anonymous closure> (dart:async/future.dart:361)
#10     _RootZone.runUnaryGuarded (dart:async/zone.dart:1093)
#11     _RootZone.bindUnaryCallback.<anonymous closure> (dart:async/zone.dart:1122)
#12     _RootZone.runUnary (dart:async/zone.dart:1155)
#13     _Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:484)
#14     _Future._propagateToListeners (dart:async/future_impl.dart:567)
#15     _Future._completeWithValue (dart:async/future_impl.dart:358)
#16     _Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:412)
#17     _asyncRunCallbackLoop (dart:async/schedule_microtask.dart:41)
#18     _asyncRunCallback (dart:async/schedule_microtask.dart:48)
#19     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#20     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:143)

#0      _rootHandleUncaughtError.<anonymous closure> (dart:async/zone.dart:886)
#1      _asyncRunCallbackLoop (dart:async/schedule_microtask.dart:41)
#2      _asyncRunCallback (dart:async/schedule_microtask.dart:48)
#3      _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:96)
#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:143)
```

The file can be found in 
'/home/myuser/darteditor/dart-sdk/lib/internal/list.dart'
instead of 
'/home/myuser/darteditor/dart-sdk/lib/_internal/list.dart'

Where does the underline in `_internal` come from?

**Comment by zoechi** [comment]
see also http://dartbug.com/22846

**Comment by cbracken** [comment]
Resolved by 2d4cc9374165d4f503244ab703c997811f264ffe.
